src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let modifiedArray = '';
+    const lengthOfElements = this.length;
```

Comment:
`this` - is a reference to the original array we're calling this function on, so this variable could be called `arrayLength`.